BringgSDK.spec.js

```diff
@@ -434,4 +436,67 @@ describe('BringgSDK', function () {
       expect(BringgSDK.disconnect).not.toHaveBeenCalled();
     });
   });
+
+  describe('getDriverPhone', function(){
+    it('should fail if shared_uuid is not passed', function(){
+      window.$ = {get:function(){}};
+      spyOn(window.$, 'get');
```

Comment:
I added jquery in one of my pull request, so line 442 isn't needed
and I would stub `get` so you won't execute any request